contributors/design-proposals/api-proposal-design-template.md

```diff
@@ -0,0 +1,84 @@
+# Description of your Api
+
```

Comment:
Somewhere early we should call out the stage of the API the proposal would change (alpha, beta, GA), and what repo it affects.